openreview/conference/invitation.py

```diff
@@ -882,19 +887,19 @@ def __init__(self, conference, review):
             },
             'signatures': {
                 'description': 'How your identity will be displayed with the above content.',
-                'values-regex': paper_group + '/Authors'
+                'values-regex': paper_group + '/Authors|'+ conference.get_program_chairs_id()
             },
             'writers': {
                 'description': 'Users that may modify this record.',
-                'values': [conference.id, paper_group + '/Authors']
+                'values-copied': [conference.get_id(), '{signatures}']
             }
         }
 
         super(PaperReviewRebuttalInvitation, self).__init__(id = signature + '/-/' + review_rebuttal_stage.name,
             super = conference.get_invitation_id(review_rebuttal_stage.name),
             writers = [conference.id],
             signatures = [conference.id],
-            invitees = [paper_group + '/Authors', conference.support_user],
+            invitees = [paper_group + '/Authors', conference.get_program_chairs_id(), conference.support_user],
```

Comment:
I was wrong before when I asked to add the PCs to be invitees, we don't need that otherwise the PC user will have a lot of pending tasks. 

Given that PCs are writers of the invitation I think they should be able to see the buttons in the UI, if this is not happening, may be we need to change the API.